api/Window.json

```diff
@@ -260,22 +312,24 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/ondevicemotion",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true,
+              "notes": "Constructor supported. ------------- lucian95: What does that mean? See reference page. ------------"
```

Comment:
The table in the Window/ondevicemotion page is included via a macro call from another page:

    {{ page("/en-US/docs/Web/API/DeviceMotionEvent","Browser_compatibility") }}

...and that page now uses BCD, but the Window/ondevicemotion page hasn't been refreshed, so it's still showing the old data.

If we want to do the same thing here, I suppose we should not represent this separately in the data, but just include the data from web.api.DeviceMotionEvent.

But I think perhaps it might be better to separate the event itself from the objects that can omit it - a reader of Window/ondevicemotion probably cares about when *that interface* started supporting the event.

So perhaps we should represent it separately under Window, but just describe when that event is supported by Window, and omit any of the details about the event itself. But let's see what @Elchi3 thinks about cases like this :).

Reply:
Ugh, yeah, that's ... interesting. Nice finding, Will!

> a reader of Window/ondevicemotion probably cares about when that interface started supporting the event.

Yes, exactly.

> we should represent it separately in Window.json, but just describe when that event is supported by Window, and omit any of the details about the event itself.

I completely agree.